Primo.js

```diff
@@ -525,7 +547,8 @@ var testCases = [
 				"callNumber": "5552.406",
 				"language": "eng",
 				"libraryCatalog": "princeton-primo.com",
-				"publisher": "London",
+				"place": "London",
+				"publisher": "London 1860-1912",
```

Comment:
Not sure if the publisher here is better or worse, but the new parsing seems overall reasonable and cutting off too much is risky.